### ISSUE TITLE
Scale from center option

### ISSUE THREAD

**Issue by Trainmaster2** [rocksdanister/lively#334]
**Is your feature request related to a problem? Please describe.**
I have a two monitor setup with one ultrawide and one more standard monitor. When I made my wallpaper, I made it match the ultrawide dimensions. While Windows will fill and align to the center on the normal monitor, Lively seems to fill and align to the top-left.

**Describe the solution you'd like**
A selector of some kind to change how the image is aligned. Probably just left-center-right and top-center-bottom would be good, but more precise controls could be useful to others.

**Additional context**
With Windows
![image](https://user-images.githubusercontent.com/18447330/109405356-a9d4a100-7924-11eb-9b49-f61a2597afee.png)
With Lively
![image](https://user-images.githubusercontent.com/18447330/109405345-8c073c00-7924-11eb-931c-bb11e7f5707d.png)

**Comment by rocksdanister** [comment]
![Screenshot 2021-02-28 082930](https://user-images.githubusercontent.com/17554161/109406438-17bdae80-799f-11eb-9973-722ed5030862.jpg)

Have you tried changing the scaler in `Choose a Fit` ? Uniform should work in this case.

**Comment by Trainmaster2** [comment]
The Lively image was actually using Uniform Fill. Uniform adds margins to the top and bottom and Fill squishes the image.

**Comment by rocksdanister** [comment]
Can you share the file?

**Comment by Trainmaster2** [comment]
Here you go.
[
![text background](https://user-images.githubusercontent.com/18447330/109406638-90852200-792f-11eb-8d23-fed5f7794d26.png)
](url)

**Comment by rocksdanister** [comment]
That 20720x8741 resolution crashed mpv player 🙄
I did not know that windows did scaling differently, yeah this looks like a useful feature to have.
This is somewhat related to this https://github.com/rocksdanister/lively/issues/311

**Comment by Trainmaster2** [comment]
Yeah, sorry about the resolution. I know it's crazy high, but more is better, right?
